aptible/client_utils_test.go

```diff
@@ -69,15 +70,15 @@ func TestGenerateErrorFromClientError(t *testing.T) {
 				},
 			},
 			wantErr:   true,
-			errorBody: "unable to properly decode error (missing fields to properly generate error) status code (400) error message (resource not found)",
+			errorBody: "unable to properly decode error (missing fields to properly generate error) -  status code (400) error message (resource not found)\n",
 		},
 		{
-			name: "return a marshalable error (if server sends back garbled response)",
+			name: "return a marshalable error with a nil payload should break early",
```

Comment:
This is not longer a "marshalable error" it's an "invalid payload" error.

Reply:
this one is marshalable but is nil actually, discussed on pair